packages/core/src/lib/TwaGenerator.ts

```diff
@@ -20,8 +20,8 @@ import fetch from 'node-fetch';
 import {template} from 'lodash';
 import {promisify} from 'util';
 import {TwaManifest, ShortcutInfo} from './TwaManifest';
-import Jimp = require('jimp');
 import Log from './Log';
+const sharp = require('sharp');
```

Comment:
Sharp has Typescript types: https://www.npmjs.com/package/@types/sharp. We can install it with `npm install --save @types/sharp` in the packages/core folder.

The, we can use a proper import statement here (and also get type checking in the code)